_layouts/post.html

```diff
@@ -28,9 +28,9 @@ <h1> {{ page.title }} </h1>
 
   <div align="center">
     {% include share.html %}
+    {% include openring_template_out.html %}
```

Comment:
I think we need an `if` statement here that checks if this file exists before including it.

Reply:
I just made this change and I created this instruction inside the openring_hook.rb file.